src/Tools/Performance Toolkit/App/src/BCPTLine.Codeunit.al

```diff
@@ -164,13 +164,14 @@ codeunit 149005 "BCPT Line"
     var
         BCPTLogEntry: Record "BCPT Log Entry";
         BCPTRoleWrapperImpl: Codeunit "BCPT Role Wrapper"; // single instance
-        IsHandled: Boolean;
         ValuesAreChanged: Boolean;
         BCPTTestSuite: Codeunit "BCPT Test Suite";
+        ModifiedMessage, ModifiedOperation : Text;
+        ModifiedExecutionSuccess: Boolean;
     begin
-        BCPTTestSuite.OnBeforeBCPTLineAddLogEntry(BCPTLine."BCPT Code", BCPTLine."Codeunit ID", BCPTLine.Description, Operation, ExecutionSuccess, Message, IsHandled, ValuesAreChanged);
-        if IsHandled then
-            exit;
+        InitValuesSubscriberCanModify(Operation, ExecutionSuccess, Message, ModifiedExecutionSuccess, ModifiedMessage, ModifiedOperation);
+        BCPTTestSuite.OnBeforeBCPTLineAddLogEntry(BCPTLine."BCPT Code", BCPTLine."Codeunit ID", BCPTLine.Description, Operation, ExecutionSuccess, Message, ModifiedOperation, ModifiedExecutionSuccess, ModifiedMessage);
+        UpdateValuesFromSubscribers(Operation, ExecutionSuccess, Message, ModifiedExecutionSuccess, ModifiedMessage, ModifiedOperation);
```

Comment:
why not (simply)
OriginalOperation := Operation;
OriginalMessage := Message;
OriginalExecutionSuccess := ExecutionSuccess;
BCPTTestSuite.OnBeforeBCPTLineAddLogEntry(BCPTLine."BCPT Code", BCPTLine."Codeunit ID", BCPTLine.Description, OriginalOperation, OriginalExecutionSuccess, OriginalMessage, Operation, ExecutionSuccess, Message);

Reply:
this does the same and you do not need to compare the values afterwards.

Reply:
Bah! 🤦‍♂️ That is the third time I make change, test, it doesn't work, fix the error and then get feedback that would have been better to refactor the change instead of fixing the error. I need star thinking of the complete solution instead of just fixing the error 🤪
Fix done